manticore/platforms/evm.py

```diff
@@ -769,9 +769,16 @@ def extend_with_zeroes(b):
             except Exception as e:
                 return
 
+        prev_was_jumpi = False
         for i in EVMAsm.disassemble_all(extend_with_zeroes(bytecode)):
             if i.mnemonic == "JUMPDEST":
                 self._valid_jumpdests.add(i.pc)
+            if prev_was_jumpi:
```

Comment:
_valid_jumpdests is supposed to be the lists of valid jumpdests. Just taking out the appearences of jumpdests iopcodes that are inside a push.